src/data/sym/basic.lean

```diff
@@ -287,8 +287,8 @@ lemma map_injective {β : Type*} {f : α → β} (hf : injective f) (n : ℕ) :
 def equiv_congr {α β : Type*} (e : α ≃ β) : sym α n ≃ sym β n :=
 { to_fun := map e,
   inv_fun := map e.symm,
-  left_inv := λ x, by rw [map_map, equiv.symm_comp_self, map_id],
-  right_inv := λ x, by rw [map_map, equiv.self_comp_symm, map_id] }
+  left_inv := λ x, by simp,
```

Comment:
@arthurpaulino, this is a regression IMO. The rewrites were still one-liners.

Reply:
They stopped working once I fixed the lint errors. Let me do better, just a sec